package.json

```diff
@@ -123,7 +123,7 @@
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "file-loader": "^0.11.1",
+    "file-loader": "^6.2.0",
```

Comment:
# Upgrade `file-loader` to the latest version 6.2.0

```
[frontend] ✖ ｢wdm｣: Entrypoint app-main = vendors-node_modules_css-loader_lib_css-base_js-node_modules_css-loader_lib_url_escape_js-nod-a5171a.hot.bundle.js?42dabef00344531a7f2f vendors-node_modules_ace-builds_src-min-noconflict_ace_js-node_modules_ace-builds_src-min-noc-2e5415.hot.bundle.js?a07e666296aee02b9af3 vendors-node_modules_babel_standalone_babel_js-node_modules_babel-preset-react_lib_index_js-n-e9bac3.hot.bundle.js?84b2e0e8efd2ee668d7f app_js-reducers-common_js-node_modules_moment-timezone_node_modules_moment_locale_sync_recurs-4e241d.hot.bundle.js?38d14bcb723ecb0a6ff6 app-main.hot.bundle.js?7c8bcf9cca422045e832 24 auxiliary assets
[frontend] Entrypoint app-public = vendors-node_modules_css-loader_lib_css-base_js-node_modules_css-loader_lib_url_escape_js-nod-a5171a.hot.bundle.js?42dabef00344531a7f2f vendors-node_modules_ace-builds_src-min-noconflict_ace_js-node_modules_ace-builds_src-min-noc-2e5415.hot.bundle.js?a07e666296aee02b9af3 app_js-reducers-common_js-node_modules_moment-timezone_node_modules_moment_locale_sync_recurs-4e241d.hot.bundle.js?38d14bcb723ecb0a6ff6 reducers-public_js-public_containers_PublicApp_jsx.hot.bundle.js?8e7a53ff75be8bff46e7 app-public.hot.bundle.js?b1df2b90da3c973642e3 24 auxiliary assets
[frontend] Entrypoint app-embed = vendors-node_modules_css-loader_lib_css-base_js-node_modules_css-loader_lib_url_escape_js-nod-a5171a.hot.bundle.js?42dabef00344531a7f2f vendors-node_modules_ace-builds_src-min-noconflict_ace_js-node_modules_ace-builds_src-min-noc-2e5415.hot.bundle.js?a07e666296aee02b9af3 app_js-reducers-common_js-node_modules_moment-timezone_node_modules_moment_locale_sync_recurs-4e241d.hot.bundle.js?38d14bcb723ecb0a6ff6 reducers-public_js-public_containers_PublicApp_jsx.hot.bundle.js?8e7a53ff75be8bff46e7 app-embed.hot.bundle.js?a3d022f0f923eb8196bc 24 auxiliary assets
[frontend] Entrypoint styles = vendors-node_modules_css-loader_lib_css-base_js-node_modules_css-loader_lib_url_escape_js-nod-a5171a.hot.bundle.js?42dabef00344531a7f2f styles.hot.bundle.js?3b11b47203fd50bcaf20 21 auxiliary assets
[frontend] 
[frontend] WARNING in DefinePlugin
[frontend] Conflicting values for 'process.env.NODE_ENV'
[frontend] 
[frontend] 1 warning has detailed information that is not shown.
[frontend] Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.
[frontend] 
[frontend] ERROR in ../../../node_modules/babel-plugin-transform-react-display-name/lib/index.js 95:12-27
[frontend] Module not found: Error: Can't resolve 'path' in '/Users/offbynone/code/work/metabase/node_modules/babel-plugin-transform-react-display-name/lib'
[frontend] 
[frontend] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[frontend] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[frontend] 
[frontend] If you want to include a polyfill, you need to:
[frontend]      - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
[frontend]      - install 'path-browserify'
[frontend] If you don't want to include a polyfill, you can use an empty module like this:
[frontend]      resolve.fallback: { "path": false }
[frontend]  @ ../../../node_modules/babel-preset-react/lib/index.js 17:44-96
[frontend]  @ ./internal/components/ScratchApp.jsx 40:24-53
[frontend]  @ ./internal/components/ sync (\w+)App.jsx$ ./ScratchApp.jsx
[frontend]  @ ./internal/routes.js 46:10-80
[frontend]  @ ./routes.jsx 465:24-67
[frontend]  @ ./app-main.js 7:14-40
[frontend] 
[frontend] ERROR in ../../../node_modules/password-generator/lib/password-generator.js 99:18-47
[frontend] Module not found: Error: Can't resolve 'crypto' in '/Users/offbynone/code/work/metabase/node_modules/password-generator/lib'
[frontend] 
[frontend] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[frontend] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[frontend] 
[frontend] If you want to include a polyfill, you need to:
[frontend]      - add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
[frontend]      - install 'crypto-browserify'
[frontend] If you don't want to include a polyfill, you can use an empty module like this:
[frontend]      resolve.fallback: { "crypto": false }
[frontend]  @ ../../../node_modules/password-generator/index.js 1:0-52
[frontend]  @ ./lib/utils.js 20:25-54
[frontend]  @ ./lib/settings.js 29:13-42
[frontend]  @ ./app.js 48:16-48
[frontend]  @ ./app-main.js 5:11-34
[frontend] 
[frontend] ERROR in ../../../node_modules/vfile/core.js 3:11-26
[frontend] Module not found: Error: Can't resolve 'path' in '/Users/offbynone/code/work/metabase/node_modules/vfile'
[frontend] 
[frontend] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[frontend] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[frontend] 
[frontend] If you want to include a polyfill, you need to:
[frontend]      - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
[frontend]      - install 'path-browserify'
[frontend] If you don't want to include a polyfill, you can use an empty module like this:
[frontend]      resolve.fallback: { "path": false }
[frontend]  @ ../../../node_modules/vfile/index.js 4:12-32
[frontend]  @ ../../../node_modules/unified/index.js 6:12-28
[frontend]  @ ../../../node_modules/react-markdown/lib/react-markdown.js 6:14-32
[frontend]  @ ./visualizations/visualizations/Text.jsx 38:21-46
[frontend]  @ ./visualizations/register.js 47:12-44
[frontend]  @ ./app.js 28:16-59
[frontend]  @ ./app-main.js 5:11-34
[frontend] 
[frontend] ERROR in ../../../node_modules/vfile/node_modules/replace-ext/index.js 3:11-26
[frontend] Module not found: Error: Can't resolve 'path' in '/Users/offbynone/code/work/metabase/node_modules/vfile/node_modules/replace-ext'
[frontend] 
[frontend] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[frontend] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[frontend] 
[frontend] If you want to include a polyfill, you need to:
[frontend]      - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
[frontend]      - install 'path-browserify'
[frontend] If you don't want to include a polyfill, you can use an empty module like this:
[frontend]      resolve.fallback: { "path": false }
[frontend]  @ ../../../node_modules/vfile/core.js 4:14-36
[frontend]  @ ../../../node_modules/vfile/index.js 4:12-32
[frontend]  @ ../../../node_modules/unified/index.js 6:12-28
[frontend]  @ ../../../node_modules/react-markdown/lib/react-markdown.js 6:14-32
[frontend]  @ ./visualizations/visualizations/Text.jsx 38:21-46
[frontend]  @ ./visualizations/register.js 47:12-44
[frontend]  @ ./app.js 28:16-59
[frontend]  @ ./app-main.js 5:11-34
[frontend] 
[frontend] 4 errors have detailed information that is not shown.
[frontend] Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.
[frontend] 
[frontend] webpack 5.37.0 compiled with 4 errors and 1 warning in 1966 ms
[frontend] ℹ ｢wdm｣: Failed to compile.
```

Further reading:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed

> MIGRATION:

> - Try to use frontend-compatible modules whenever possible.
> - It's possible to manually add a polyfill for a Node.js core module. An error message will give a hint on how to achieve that.
> - Package authors: Use the browser field in package.json to make a package frontend-compatible. Provide alternative implementations/dependencies for the browser.